test/TestUtilities/Test.Utility/ProjectManagement/TestProjectKProject.cs

```diff
@@ -69,4 +69,124 @@ public Task UninstallPackageAsync(INuGetPackageMoniker package, IReadOnlyDiction
             return Task.Run(() => { _installedPackages.RemoveAll(p => p.Id == package.Id && p.Version == package.Version); });
         }
     }
+
+    /// <summary>
+    /// Represents metadata about a nuget package.
+    /// </summary>
+    [ComImport]
+    [Guid("BCA2E197-E352-4474-B0A7-D8CE606CD4E9")]
+    public interface INuGetPackageMoniker
```

Comment:
Note to self: Maybe place these in a different file and explain why they're needed?